### ISSUE TITLE
Missing validation dataset for some benchmark tasks

### ISSUE THREAD

**Issue by BSharmi** [nadavbra/protein_bert#24]
Hello,

I downloaded the benchmark dataset but noticed that for some tasks such as SCOP the validation csv file is missing. Can I get access to the files please?

Thank you!

**Comment by nadavbra** [comment]
Not all benchmarks have a validation set. In these cases, you can take a subset of the training set and use it as a validation set.
